net/quagga/src/opnsense/mvc/app/controllers/OPNsense/Quagga/Api/DiagnosticsController.php

```diff
@@ -59,6 +59,12 @@ public function showipbgpsummaryAction()
         $response = $backend->configdRun("quagga diag-bgp summary");
         return array("response" => $response);
     }
+    public function showrunningconfigAction()
+    {
+        $backend = new Backend();
+        $response = $backend->configdRun("quagga general-runningconfig");
+        return array("response" => $response);
+    }
```

Comment:
Do you mean like this: https://github.com/mimugmail/plugins/blob/shrun2/net/quagga/src/opnsense/mvc/app/controllers/OPNsense/Quagga/Api/DiagnosticsController.php#L128-L131 
I always get a "null" in the UI